region/brinstar/pink/Pink Brinstar Hopper Room.json

```diff
@@ -355,7 +362,11 @@
       "name": "Spring Ball from the Doorframe",
       "requires": [
         {"doorUnlockedAtNode": 1},
-        "h_canMaxHeightSpringBallJump",
+        "canTrickySpringBallJump",
+        {"or":[
+          "canTrickyJump",
+          "canLateralMidAirMorph"
```

Comment:
Its not really a lateralMidAirMorph.  A delayed morph, or an early morph and springfling movement boost could do it.  But lateralmidairmorph works as an alternative to a trickyJump 🤷